language-extensions/java/src/linux/JavaExtensionUtils_linux.cpp

```diff
@@ -163,6 +163,25 @@ JavaExtensionUtils::fn_createJvm JavaExtensionUtils::LoadJvm(const string& jvmPa
 	return reinterpret_cast<fn_createJvm>(dlsym(g_jvmLib, "JNI_CreateJavaVM"));
 }
 
+//----------------------------------------------------------------------------
+// Name: JavaExtensionUtils::ShutdownJvm
+//
+// Description:
+//  Cleans up JVM resources. Doesn't call Invocation API function
+//  DestroyJavaVM since ExtHost process exit defers to OS for resource
+//  cleanup and SPEES query doesn't hang.
+//  Follow up investigation tracked via work item: 3090980
+//
+void JavaExtensionUtils::ShutdownJvm(JavaVM *jvm)
+{
+	LOG("Cleaning up JVM");
+
+	if (jvm != nullptr)
```

Comment:
nit: just noticed it; it's not incorrect but the condition seems useless; if it's !nullptr, we make it; so, we could as well always set it to nullptr.

Reply:
done